packages/pigeon/example/app/lib/src/messages.g.dart

```diff
@@ -130,7 +130,7 @@ class ExampleHostApi {
     }
   }
 
-  Future<int> add(int arg_a, int arg_b) async {
+  Future<int> add(int a, int b) async {
```

Comment:
That's a fair point, I would prefer if people could use their defined names in these instances, since they actually matter now. I could try to maintain a list of reserved keywords to filter out. That does add complexity and make some rules more hidden though...

Reply:
Actually, I think we should change the generated variable names to be less likely to collide and let the user do what they want. `_$_$channelName` or something even even less collide-able like a hashed string or something. I don't this option a reduces readability, and I'm not sure if it matters too much even if it does.

Thoughts?

Reply:
Oh right, I forgot this matters now with named parameters. Complex names for our own that nobody would ever reasonably use with asserts just in case SGTM.

The use of `$` at the start kind of makes my eyes bleed; I didn't even know that was a valid identifier. Maybe `__pigeon_*`, and we turn of the lint about leading underscores for local variables?

Reply:
That doesn't work for the codec specifically, since that is a public static member

Reply:
I can just do something different for codec only